connectors/src/connectors/microsoft/temporal/activities.ts

```diff
@@ -520,15 +520,27 @@ export async function syncDeltaForRootNodesInDrive({
         ...sortForIncrementalUpdate(uniqueChangedItems, rootNode.id)
       );
     });
+
+    await scrubRemovedFolders({
+      connector,
+      uniqueChangedItems,
+      sortedChangedItems,
+    });
   }
 
-  // Finally add all removed items, which may not have been included even if they are in the selected roots
+  // @thomas - I don't get that code block below; the deleted items should
+  // already be in sortedChangedItems if they're in the selected roots, right?
+  // maybe it's legacy and we can now delete it?
+
+  // Finally add all removed items, which may not have been included even if they are in
+  // the selected roots
```

Comment:
Oh, I think I get it, this was an alpha version of scrubbing the "removed" folder (only for those that were deleted after being removed)
So it would be safe to remove. Correct?

Reply:
yes, safe to remove, the case is now handled in scrubRemovedFolders